cryton/worker/empire.py

```diff
@@ -230,7 +230,7 @@ async def deploy_agent(arguments: dict) -> dict:
     if session_id:
         metasploit_obj = MetasploitClientUpdated()
         try:
-            session_to_use = metasploit_obj.sessions.get(session_id)
+            session_to_use = metasploit_obj.sessions.get(int(session_id))
```

Comment:
Once the validation is updated, please revert this change.

Reply:
Addressed in [dc336a6](https://github.com/CSIRT-MU/Cryton/pull/1/commits/dc336a6e825faeb503e169d4885a43a21ab3d08f), thanks for the input!